x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/details/overview/index.tsx

```diff
@@ -4,91 +4,95 @@
-          {
-            defaultMessage: 'Overview',
-          }
-        ),
-        content: (
-          <>
-            <EuiSpacer />
-            <MetadataPanel />
-          </>
-        ),
-      },
-      {
-        id: 'overviewResolver',
-        name: i18n.translate(
-          'xpack.endpoint.application.endpoint.alertDetails.overview.tabs.resolver',
-          {
-            defaultMessage: 'Resolver',
-          }
-        ),
-        content: (
-          <>
-            <EuiSpacer />
-            {selectedAlertIsLegacyEndpointEvent && <AlertDetailResolver />}
-          </>
-        ),
-      },
-    ];
-  }, [selectedAlertIsLegacyEndpointEvent]);
+    const tabs = useMemo(() => {
+      return [
+        {
+          id: 'overviewMetadata',
+          name: i18n.translate(
+            'xpack.endpoint.application.endpoint.alertDetails.overview.tabs.overview',
+            {
+              defaultMessage: 'Overview',
+            }
+          ),
+          content: (
+            <>
+              <EuiSpacer />
+              <MetadataPanel />
+            </>
+          ),
+        },
+        {
+          id: 'overviewResolver',
+          name: i18n.translate(
+            'xpack.endpoint.application.endpoint.alertDetails.overview.tabs.resolver',
+            {
+              defaultMessage: 'Resolver',
+            }
+          ),
+          content: (
+            <>
+              <EuiSpacer />
+              <AlertDetailResolver selectedEvent={(alertDetailsData as unknown) as ResolverEvent} />
+            </>
+          ),
+        },
+      ];
+    }, [alertDetailsData]);
 
-  return (
-    <>
-      <section className="details-overview-summary">
-        <EuiTitle size="s">
-          <h3>
-            <FormattedMessage
-              id="xpack.endpoint.application.endpoint.alertDetails.overview.title"
-              defaultMessage="Detected Malicious File"
-            />
-          </h3>
-        </EuiTitle>
-        <EuiSpacer />
-        <EuiText>
-          <p>
-            <FormattedMessage
-              id="xpack.endpoint.application.endpoint.alertDetails.overview.summary"
-              defaultMessage="MalwareScore detected the opening of a document on {hostname} on {date}"
-              values={{
-                hostname: alertDetailsData.host.hostname,
-                date: <FormattedDate timestamp={alertDetailsData['@timestamp']} />,
-              }}
-            />
-          </p>
-        </EuiText>
-        <EuiSpacer />
-        <EuiText>
-          Endpoint Status: <EuiHealth color="success">Online</EuiHealth>
-        </EuiText>
-        <EuiText>Alert Status: Open</EuiText>
-        <EuiSpacer />
-      </section>
-      <EuiTabbedContent tabs={tabs} initialSelectedTab={tabs[0]} />
-    </>
-  );
-});
+    return (
+      <>
+        <section className="details-overview-summary">
+          <EuiTitle size="s">
+            <h3>
```

Comment:
❔ I think the h3 might be ( ❔ ) redundant but maybe you have to use the `size="l"` variant of EUITitle